### PR TITLE
CI: add script to create bundles for microshift

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

**Comment by praveenkumar** [comment]
Required for https://github.com/openshift/release/pull/38200 one.

File: ci_microshift.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+sudo yum install -y podman make golang rsync
```
Comment (anjannath) [inline ci_microshift.sh:5]: podman is also installed by the `microshift.sh` script, might not be needed here? https://github.com/crc-org/snc/blob/cfe2c7dc246fd8efb769a6cd0bdbb80d7280d3b0/microshift.sh#L57

**Review by anjannath — APPROVED** [review]

File: ci_microshift.sh (hunk 1 above)
Comment (anjannath) [reply ci_microshift.sh:5]: also `rsync` is not being used by the script?

File: ci_microshift.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+./shellcheck.sh
+./microshift.sh
+
+# Run createdisk script
```
Comment (cfergeau) [inline ci_microshift.sh:10]: If you add a comment, it's more useful to document why we use these zstd flag

File: ci_microshift.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+set -exuo pipefail
+
+sudo yum install -y make golang
+
+./shellcheck.sh
+./microshift.sh
+
+# Run createdisk script
+# Set the zstd compression level to 10 to have faster
+# compression.
```
Comment (cfergeau) [inline ci_microshift.sh:12]: There's a `--fast` option if the goal is to get fast compression. You could also drop `--long`?

File: ci_microshift.sh [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+export CRC_ZSTD_EXTRA_FLAGS="-10 --long"
+./createdisk.sh crc-tmp-install-data
+
+git clone https://github.com/code-ready/crc.git
```
Comment (gbraad) [inline ci_microshift.sh:16]: Perhaps add comment why we run with latest of crc.

**Review by gbraad — APPROVED** [review]
Latest is a clear choice. Worth adding comment.
Otherwise /lgtm

File: ci_microshift.sh (hunk 4 above)
Comment (cfergeau) [reply ci_microshift.sh:16]: *grin* `/code-ready/`?

File: ci_microshift.sh (hunk 3 above)
Comment (praveenkumar) [reply ci_microshift.sh:12]: with fast the bundle size would be not that optimized so using this is kind of proper balance the size of bundle and time taken.

**Pull request merged**